docs/src/prototypes/IconViewer/index.tsx

```diff
@@ -1,14 +1,43 @@
 import * as React from 'react'
 import { Provider, Grid, Divider, Header, Icon } from '@stardust-ui/react'
 import themeWithProcessedIcons from 'src/themes/teams/withProcessedIcons'
+import { TeamsProcessedSvgIconSpec } from 'src/themes/teams/components/Icon/svg/types'
+
+import { Menu } from 'semantic-ui-react'
 
 const cellStyles = {
   margin: '10px 0',
 }
 
 const processedIconsNamePrefix = 'processedIcons_'
 
+const renderStardustIconName = (icon, isOutline = false) => {
+  const maybeExportedAs = (icon as any).exportedAs
+  return (
+    maybeExportedAs && (
+      <code style={{ color: 'red' }}>
+        => {maybeExportedAs} {isOutline && 'outline'}
+      </code>
+    )
+  )
+}
+
 class IconViewerExample extends React.Component<any, {}> {
+  private readonly iconFilters = {
+    All: () => true,
+    Exported: (icon: TeamsProcessedSvgIconSpec) => icon.exportedAs,
+    'Not Exported': (icon: TeamsProcessedSvgIconSpec) => !icon.exportedAs,
+  }
```

Comment:
This object doesn't need to be a class property, it doesn't reference `this` nor the class.

Reply:
I would see this object still being part of the class, as this is the only place where this one is necessary. Thus, to make the class itself and this object being cohesive, I would rather leave this object here, even if it is a `static` one by its semantics

Reply:
Since this is a subjective judgement call and only prototype code, I'll agree to move us forward.

If it were production code, I would make a case for why you should only implement required code and nothing more.